src/packages/@ncigdc/components/Explore/GenesTab.js

```diff
@@ -77,10 +78,13 @@ export default compose(
       );
       push({
         pathname: '/exploration',
-        query: {
-          ...query,
-          filters: JSURL.stringify(newFilters),
-        },
+        query: _.omitBy(
```

Comment:
Maybe use `removeEmptyKeys` from utils/uri. It removes more then just undefined and null;